core/src/control/precice/nested_solver_fast_monodomain_solver.tpp

```diff
@@ -581,7 +581,11 @@ void PreciceAdapterNestedSolver<FastMonodomainSolver<T1>>::getTractionValues(
 
 template <typename T1>
 Vec PreciceAdapterNestedSolver<FastMonodomainSolver<T1>>::currentState(
-    NestedSolverType &nestedSolver) {}
+    NestedSolverType &nestedSolver) {
+  LOG(ERROR) << "Implicit coupling not supported! curentState is not "
+                "implemented for this preCICE adapter nested solver";
+  return nullptr;
```

Comment:
I assume there is nothing better to return here, but it also does not matter, as long as it exits above.